source/jsonnet/england-wales/lib/placeholders.libsonnet

```diff
@@ -66,4 +66,62 @@
       },
     }],
   },
+  ordinal: {
+    placeholder: 'ordinal',
```

Comment:
These could be:
- `household_ordinality`
- `visitor_ordinality`
- `household_cardinality`
- `visitor_cardinality`

Thoughts?

Reply:
I think `ordinality` is the correct term to use here. It is a key which references a value that is the ordinality of the household that is itself an ordinal number.

Reply:
so `household_ordinality` etc

Reply:
Done

Reply:
Thanks, added!